File: LeetCode/Easy/Find Indices of Stable Mountains/3285.js
/**
 * @param {number[]} height
 * @param {number} threshold
 * @return {number[]}
 */
var stableMountains = function(height, threshold) {
    const resultList = [];

    for (let i = 1; i < height.length; i++) {
        if (height[i - 1] > threshold) {
            resultList.push(i);
        }
    }

    return resultList;
};

File: LeetCode/Easy/Find Indices of Stable Mountains/3285.java
class Solution {
    public List<Integer> stableMountains(int[] height, int threshold) {
        List<Integer> resultList = new ArrayList<>();
        
        for (int i = 1; i < height.length; i++) {
            if (height[i - 1] > threshold) {
                resultList.add(i);
            }
        }

        return resultList;
    }
}
